public/content/lessons/2017/taylor-series/index.mdx

```diff
@@ -7,4 +7,376 @@ video: 3d6DsjIBzJ4
+/>
+
+This is true for any input, no matter how far away from $0$ it is, even though these Taylor polynomials are constructed only from derivative information gathered at the input $0$.
+In a case like this, we say $e^x$ equals its Taylor series at all inputs $x$, which is kind of a magical thing to have happen.
+
+## Limitations for $\ln(x)$
+
+Although this is also true for some other important functions, like sine and cosine, sometimes these series only converge within a certain range around the input whose derivative information you're using.
+
+If you work out the Taylor series for the natural log of $x$ around the input $x = 1$, which is built from evaluating the higher order derivatives of ln(x) at x=1, this is what it looks like.
+
+<Figure
+    image="figures/natural-logarithm-derivatives-and-taylor-polynomial.png" 
+/>
+
+When you plug in an input between $0$ and $2$, adding more and more terms of this series will indeed get you closer and closer to the natural log of that input.
+
+<Figure
+    image="figures/natural-logarithm-taylor-series-input-between-0-and-2.png" 
+/>
+
+But outside that range, even by just a bit, the series fails to approach anything.
+
+As you add more and more terms the sum bounces back and forth wildly, it does not approaching the natural log of that value, even though the natural log of $x$ is perfectly well defined for inputs above $2$.
+
+<Figure
+    image="figures/natural-logarithm-taylor-series-input-above-2.png" 
+/>
+
+In some sense, the derivative information of $\ln(x)$ at $x=1$ doesn't propagate out that far. In a case like this, where adding more terms of the series doesn't approach anything, you say the series **diverges**.
+
+And that maximum distance between the input you're approximating near, and points where the outputs of these polynomials actually do converge, is called the "radius of convergence" for the Taylor series.
+
+<Figure
+    image="figures/natural-logarithm-radius-of-convergence.png" 
+/>
+
+## Summary
+
+There remains more to learn about Taylor series, their many use cases, tactics for placing bounds on the error of these approximations, tests for understanding when these series do and don't converge. For that matter there remains more to learn about calculus as a whole, and the countless topics not touched by this series.
+
+The goal with these videos is to give you the fundamental intuitions that make you feel confident and efficient learning more on your own, and potentially even rediscovering more of the topic for yourself. In the case of Taylor series, the fundamental intuition to keep in mind as you explore more is that they translate derivative information at a single point to approximation information around that point.
+
+TODO: maybe add "and in the case of the exponential function where the series converges for all values, this can be used as a definition of the exponential function throughout other areas of math."
```

Comment:
Maybe!  If there was ever an article on the matrix exponents video, we could also just link out to that.